src/Components/DoubleText.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {AbsoluteFill} from 'remotion';
+
+export const DoubleText: React.FC<{
```

Comment:
I think that this component is not necessary, it's just the display of two texts that don't necessarily need to be in a component that you can customise 😊

Reply:
Removed it 👍